Testscripts/Linux/docker_python_app.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+########################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# Description:
+#   Script to run the hello-world python app in docker.
+#
+########################################################################
+
+DOCKER_FILENAME="Dockerfile"
+
+PYTHON_PROG_NAME="helloworld.py"
+
+CONTAINER_NAME="pythonapp"
+CONTAINER_IMAGE_NAME="pythonappimage"
+
+STRING_IDENTIFIER="Hello world from python"
+
+# Function to generate python program which will run
+# inside docker
+GeneratePythonProgram()
```

Comment:
GeneratePythonProgram() {

We want to keep the consistent format. Please change all.

Reply:
Fixed.